corehq/apps/app_manager/templates/app_manager/v1/form_view_base.html

```diff
@@ -259,6 +259,10 @@
             $('#cloudcare-preview-url').click(function(e) {
                 ga_track_event('CloudCare', 'Click "Preview Form"');
                 analytics.workflow("Clicked Preview Form");
+                {% if request.couch_user.days_since_created == 0 %}
+                    ga_track_event('CloudCare', 'Click "Preview Form" within first 24 hours');
+                    analytics.workflow("Clicked Preview Form within first 24 hours");
```

Comment:
Tweaked the names to make them match, for the sake of sanity. I don't remember why we agreed to have both a kiss event and a GA event - I think the major benefit of kiss here is being able to view the data as a funnel, and the benefit of GA is being able to look at the data by session.